server/templates/pages/src/style-guide.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="referrer" content="origin">
+        <meta name="robots" content="noindex,nofollow">
+        <ink rel="stylesheet" href="/styles/main.css">
+    </head>
+    <body>
+        <label class="mdl-checkbox mdl-js-checkbox" for="checkbox-1">
+          <input type="checkbox" id="checkbox-1" class="checkbox_test">
```

Comment:
so currently the style guide looks like this:

![](http://v14d.com/i/586bf9f247de7.png)

See if you can make it show the proper checkbox, such as this one: 
*****
![image](https://cloud.githubusercontent.com/assets/128755/21619992/2d5c9930-d1c0-11e6-8e76-04fdaebba216.png)
*****
 (from the /signup screen)

and a radio group that is proposed in the issue https://github.com/mozilla/fxa-content-server/issues/4303